src/components/Grid/Grid.story.js

```diff
@@ -19,21 +19,20 @@ import styled from '@emotion/styled';
 import Grid from './Grid';
 import Col from '../Col';
 import Row from '../Row';
-import { circuit } from '../../themes';
 
 const StyledCol = styled(Col)`
-  color: ${circuit.colors.white};
+  color: ${p => p.theme.colors.white};
```

Comment:
Why not destructure here? (I think it's what we do in most components)

```suggestion
  color: ${({ theme }) => theme.colors.white};
```

Reply:
I find the destructuring pattern more tedious and longer to type, so I personally favor the pattern I used. We don't have a company-convention here — the partner portal codebase uses the `p => p.theme...` pattern and I take it that you use the `({ theme }) => theme...` pattern in the dashboard?

Reply:
Well, we are using the destructuring pattern in Circuit UI. You're right, I should stick to that until we decide to adopt a different convention.

Reply:
Makes sense! I do agree that `p => p.theme` makes it faster to type 🙂  Let's look at harmonizing this in the future! There might even be some way to use ESLint for this.